src/Stratis.Bitcoin/Base/TimeSyncBehavior.cs

```diff
@@ -297,6 +291,31 @@ private void RecalculateTimeOffsetLocked()
             this.logger.LogTrace("(-):{0}={1}", nameof(this.timeOffset), this.timeOffset.TotalSeconds);
         }
 
+        /// <summary>
+        /// Gets the median offset of all the inbound samples 
+        /// limited to 1 less than the <see cref="OutboundToInboundWeightRatio"/> inbound per outbound) 
+        /// and the outbound samples with a weighting of the OutboundToInboundWeightRatio.
+        /// </summary>
+        /// <returns>Median offset in seconds.</returns>
+        private double GetMedianOffsetInSecondsFromInboundAndWeightedOutbound()
+        {
+            CircularArray<TimestampOffsetSample> inboundOffsets = this.inboundTimestampOffsets.Select(s => s.TimeOffset.TotalSeconds);
+            CircularArray<TimestampOffsetSample> outboundOffsets = this.outboundTimestampOffsets.Select(s => s.TimeOffset.TotalSeconds);
+
+            // All samples is formed of one copy of each inbound sample
+            // and <see cref="OutboundToInboundWeightRatio"/> copies of each outbound sample.
```

Comment:
no xml in inline (`//`) comments

Reply:
ah that explains why  N was there before